File: 백준/Bronze/15372. A Simple Problem．/A Simple Problem．.c
#include<stdio.h>
#include<math.h>
int main(void){
    int t;
    scanf("%d",&t);
    for(int i = 0; i < t; i++){
        long n;
        scanf("%ld",&n);
        long k = pow(n,2);
        printf("%ld\n",k);
    }
    return 0;
}

File: 백준/Bronze/15372. A Simple Problem．/A Simple Problem．.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int T = sc.nextInt();
        for(int i = 0; i < T; i++){
            long n = sc.nextLong();
            long k = (long)Math.pow(n,2);
            System.out.println(k);
        }
    }
}